stylix/palette.nix

```
@@ -10,7 +10,10 @@ let
 
   paletteJSON = let
     generatedJSON = pkgs.runCommand "palette.json" { } ''
-      ${palette-generator}/bin/palette-generator ${cfg.polarity} ${lib.escapeShellArg cfg.image} $out
+      ${palette-generator}/bin/palette-generator \
+        "${cfg.polarity}" \
```

cc: https://github.com/danth/stylix/pull/318#pullrequestreview-1971796808

Do you mean:

```suggestion
        ${lib.escapeShellArg cfg.polarity} \
```